applications/launchpad/backend/src/docker/settings.rs

```diff
@@ -42,13 +42,13 @@ http://monero-stagenet.exan.tech:38081,\
 http://xmr-lux.boldsuck.org:38081,\
 http://singapore.node.xmr.pm:38081";
 
-#[derive(Default, Debug, Serialize, Deserialize)]
+#[derive(Default, Debug, Serialize, Deserialize, Clone)]
```

Comment:
If you don't actually clone these structs, don't bloat the codebase with the derived functions.

Reply:
pure java style :). Fixed